### ISSUE TITLE
Populate fields aren't stored correctly

### ISSUE THREAD

**Issue by KyleAMathews** [taterbase/mongoosastic#17]
I'm indexing a "posts" field that refers to an author model. The Object Id is getting mangled into something like `OÄ^úóìN`

**Comment by KyleAMathews** [comment]
The mapping elasticsearch reports is created is

```
_user: {
    dynamic: true
    properties: {
        id: {
            type: string
        }
        _bsontype: {
            type: string
        }
    }
}
```

This is odd as the mapping that Mongoosastic creates is just `_user: 'string'`

**Comment by KyleAMathews** [comment]
The problem with this turned out to be that the ObjectID wasn't being jsonified as a string. This bit of code fixed that `serializedForm._user = "" + serializedForm._user`

**Comment by KyleAMathews** [comment]
Unless you have a chance to get to this first, I'll create a pull request Saturday or Sunday to fix this and #18. In the `serialized` function I'll add a bit of code which checks for fields of types date + objectId and converts them to the formats ElasticSearch expects. I think with that, everything I need for my app will be working.

**Comment by jamescarr** [comment]
Did you issue a pull request to fix this? If not I'll make the fix now.

**Comment by KyleAMathews** [comment]
I didn't get to it on Sunday. If you're working on it right now, go ahead and fix it. I was planning on working on the Search stuff again this morning but it'll be an hour or two.

**Comment by jamescarr** [comment]
No problem... I'm going to go ahead an tackle it. Breaking the serialize function out so it can be tested directly

**Comment by jamescarr** [comment]
Fixed with f47ba4d
